src/FakeItEasy/CapturedArgument.of.TArg.TCap.cs

```diff
@@ -0,0 +1,60 @@
+namespace FakeItEasy
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Holds argument values captured from calls to a faked member.
+    /// </summary>
+    /// <typeparam name="TArg">The type of the argument.</typeparam>
+    /// <typeparam name="TCap">The type of the value to capture from the argument.</typeparam>
+    public class CapturedArgument<TArg, TCap>

```

Comment:
nit: maybe `TArgument` and `TCapture` instead of `TArg` and `TCap`?
Characters are cheap!